.github/workflows/ci.yml

```diff
@@ -31,12 +31,12 @@ jobs:
         run: npm i
       - name: Biome
         run: node --run biome:ci
+      - name: Typecheck
+        run: node --run typecheck
```

Comment:
Not sure if it's a bug with the ts eslint plugin, but `@typescript-eslint/no-unsafe-return` fails in test/website files unless the `src` project is already built 🤔